src/common/utils/setup-helpers.js

```diff
@@ -318,6 +319,24 @@ function checkNativeTextTracksSupport(options: KPOptionsObject): void {
   }
 }
 
+/**
+ * Sets config option fullscreen element for Vr Mode support
+ * @private
+ * @param {KPOptionsObject} options - kaltura player options
+ * @returns {void}
+ */
+function configureVrRelevantOptions(options: KPOptionsObject): void {
+  if (options.plugins.vr && !options.plugins.vr.disable && !options.ui.disable) {
```

Comment:
Why `inBrowserFullscreen` depending on ui?

Reply:
It's not depend, it depends for default value, there isn't reason to set default inbrowserfullscreen for ui disable as was before